File: API/DTOs/Account/InviteUserResponse.cs
namespace API.DTOs.Account;

public class InviteUserResponse
{
    /// <summary>
    /// Email link used to setup the user account
    /// </summary>
    public string EmailLink { get; set; } = default!;
    /// <summary>
    /// Was an email sent (ie is this server accessible)
    /// </summary>
    public bool EmailSent { get; set; } = default!;
    /// <summary>
    /// When a user has an invalid email and is attempting to perform a flow.
    /// </summary>
    public bool InvalidEmail { get; set; } = false;
}

File: UI/Web/src/app/_models/auth/invite-user-response.ts
export interface InviteUserResponse {
    /**
     * Link to register new user
     */
    emailLink: string;
    /**
     * If an email was sent to the invited user
     */
    emailSent: boolean;
   /**
   * When a user has an invalid email and is attempting to perform a flow.
   */
   invalidEmail: boolean;
}
